### PR TITLE
Add TripId info to markers so markers are saved to appropriate Trip

### PR DESCRIPTION
**Pull request by sukhada**
@caro5

### CODE REVIEW

File: app/src/main/java/turtler/voyageur/activities/TripActivity.java [hunk 1]
```diff
@@ -24,7 +24,9 @@ public void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_trip);
         ButterKnife.bind(this);
 
-        TripFragmentPageAdapter pagerAdapter = new TripFragmentPageAdapter(getSupportFragmentManager());
+        Bundle b = new Bundle();
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/activities/TripActivity.java]: you dont need to change tripactivity or tripfragmentpageadapter.

just do `String tripId = getActivity().getIntent().getStringExtra("tripId");` in your TripMapFragment.

the TripAdapter passes the tripId to the TripActivity, and the TripMapFragment is able to get it's parent's activity and intent through the line above.

**Comment by sukhada** [comment]
@caro5 updated

File: app/src/main/java/turtler/voyageur/adapters/TripFragmentPageAdapter.java [hunk 2]
```diff
@@ -13,6 +14,7 @@
 
 public class TripFragmentPageAdapter extends FragmentPagerAdapter {
     private String tabTitles[] = { "Timeline", "Map" };
+    private String tripId;
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/adapters/TripFragmentPageAdapter.java]: this isn't used anymore right?

File: app/src/main/java/turtler/voyageur/models/Marker.java [hunk 3]
```diff
@@ -22,6 +30,21 @@ public double getLongitudeKey() {
         return getDouble(LONGITUDE_KEY);
     }
 
+    public Trip getTrip() {
+        return (Trip) get(TRIP_KEY);
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/models/Marker.java]: should this be `return (Trip) getParseObject("trip")`?

**Comment by sukhada** [comment]
@caro5 durr sorry, updated again

**Comment by caro5** [comment]
lgtm

**Pull request merged**